TradeItIosTicketSDK2/TradeItLauncher.swift

```diff
@@ -146,8 +148,11 @@ protocol OAuthCompletionListener {
     }
 
     public func launchBrokerCenter(fromViewController viewController: UIViewController) {
-        let navController = self.viewControllerProvider.provideNavigationController(withRootViewStoryboardId: TradeItStoryboardID.brokerCenterView)
-        viewController.present(navController, animated: true, completion: nil)
+        let webViewController = self.viewControllerProvider.provideViewController(forStoryboardId: TradeItStoryboardID.webView) as! TradeItWebViewController
+        webViewController.pageTitle = "Broker Center"
+        webViewController.url = TradeItSDK.brokerCenterService.getUrl()
+
+        viewController.navigationController?.pushViewController(webViewController, animated: true)
```

Comment:
@mitochondrion what is the expectation from the `viewController` param? The `TradeItWebViewController` needs to go on to a nav controller.